src/views/Login.js

```diff
@@ -61,37 +59,54 @@ export default function Login() {
   return (
     <View pop={pop} inset>
       <Grid>
-        <Grid.Item full as={Crumbs} routes={[{ text: 'Bridge' }]} />
-        <Grid.Item full as={H4} className="mt4">
-          Login
-        </Grid.Item>
-
-        <Grid.Item
-          full
-          as={Tabs}
-          className="mt1"
-          // Tabs
-          views={VIEWS}
-          options={OPTIONS}
-          currentTab={currentTab}
-          onTabChange={setCurrentTab}
-          // Tab extra
-          goHome={goHome}
-        />
-        <Grid.Item full as={P} className="f6">
-          Version {version}
+        <Grid.Item full as={Text} className="flex justify-center mt9 mb7">
+          <Grid.Item as={Text} className="gray3">
+            Urbit ID /&nbsp;
+          </Grid.Item>
+          <Grid.Item as={Text}>Login</Grid.Item>
         </Grid.Item>
+        {isOther && <Grid.Item full as={Other} goHome={goHome} />}
+        {!isOther && <Grid.Item full as={Ticket} goHome={goHome} />}
+        {!isOther && (
+          <>
+            <Grid.Item full className="t-center mv4 gray4">
+              or
+            </Grid.Item>
+            <Grid.Item
+              full
+              as={Button}
+              className="b-solid b1 b-black"
+              center
+              onClick={() => setisOther(true)}>
+              Metamask, Mnemonic, Hardware Wallet...
+            </Grid.Item>
+            <Grid.Item full onClick={goToActivate} className="mv10 t-center f6">
+              <span className="gray4">New Urbit ID? </span>
+              <LinkButton>Activate</LinkButton>
+            </Grid.Item>
```

Comment:
Not sure why we don't just show this on both pages (ie regardless of `isOther`), but I can see how it's only strictly needed for the landing conditions.

(Also the space after "ID?" bother me. (^: )

Reply:
Yeah, this crossed my mind, but I felt like this page is for advanced users who are familiar with crypto. On the flipside, we haven't done a good job of getting people to use Master Ticket (even if they bought a planet from a third-party)